src/components/header/AboutPage.js

```diff
@@ -47,9 +47,18 @@ export default function AboutPage() {
             In its current deployment, MaterialNet provides an interactive map
             for the materials stability network (<a
             href="https://doi.org/10.1038/s41467-019-10030-5" target="_blank"
-            rel="noopener noreferrer">Aykol et al. in Nat. Comm. 2019</a>) and
-            a text co-occurrence network network extracted from <a
-            href="http://www.matscholar.com" target="_blank" rel="noopener noreferrer">MatScholar</a>.
+            rel="noopener noreferrer">Aykol et al. in Nat. Comm. 2019</a>), a
+            materials similarity network (Ward et al. Phys. Rev. B 96 (2017)),
```

Comment:
^Here we can remove the reference, as we are citing the feature vectors below already.